generator/resources/ec2_mac_test_matrix.json

```diff
@@ -2,37 +2,37 @@
       {
         "os": "macOS Big Sur Arm64",
         "instanceType":"mac2.metal",
-        "ami": "amzn-ec2-macos-11.*",
+        "ami": "cloudwatch-agent-integration-test-mac-bigsur-arm64",
```

Comment:
I thought ami builder only supported linux. I guess not

Reply:
Why could these ami not be made public?

Reply:
Image Builder does not support macOS. There are manual copy of the AMI images so I can associate them with the License Manager since we need to own the AMI to associate.

We can't make them public because we are hitting the quota for public images which is 62.